src/less/wizard-stepper/base/wizard-stepper.less

```diff
@@ -0,0 +1,129 @@
+@mobile: ~"only screen and (max-width: 600px)";
+@wizard-step-confirmation-color: #36cf57;
+@wizard-step-default-color: #767676;
+
+.wizard-stepper-list {
+    align-items: center;
+    display: flex;
+    justify-content: center;
+    list-style-type: none;
+    overflow-y: auto; // For non mobile view port widths. For mobile widths, we should fallback to vertical
+}
+
+.wizard-stepper-list__item {
+    align-items: center;
+    display: flex;
+    flex-direction: row;
+    text-decoration: none;
+}
+
+.wizard-stepper-list__item--confirmation,
+.wizard-stepper-list__item--upcoming,
+.wizard-stepper-list__item--inprogress {
+    &:extend(.wizard-stepper-list__item);
+
+    margin-right: 55px;
+}
+
+.wizard-stepper-list__item--confirmation,
+.wizard-stepper-list__item--upcoming {
+    color: @wizard-step-default-color;
+}
+
+.wizard-stepper-list__item__node {
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+    position: relative;
+    width: 30px;
+
+    &::before {
+        background-color: @wizard-step-default-color;
+        bottom: 35px;
+        content: '';
+        height: 3px;
+        left: -50px;
```

Comment:
I have a bit of a problem with this, when I looked on firefox it not aligned correctly when I loaded it initially.
<img width="527" alt="image" src="https://user-images.githubusercontent.com/1755269/74969174-e5af5f00-53d0-11ea-9fe0-e2b2e839d78c.png">


Here's my take, it might take a bit of a change but hopefully not too much.

The item__node should be flex and direction row. Afterwards the svg and text should be in it's own container.
The `:before` should have no position absolute, no left, no bottom. The flex will handle it correctly and the container.
The container with the SVG and text should have display:flex and flex-direction column. This way we let flex handle the `---` instead of doing absolute positioning which will never look correct. 

It's a bit complex. I played around with this locally and I was able to get something decent using just flex. It needs some slight work, but here's the diff that I was able to get working. (I had to upload it as a zip)
[patch.diff.zip](https://github.com/eBay/skin/files/4232538/patch.diff.zip)
You need some color changes and minor markup changes. As well as some max width. But this should be working mostly. It also aligns the text and arrows correctly (hopefully)
 
It still needs some size adjustments I believe as well as a max-width or something

However this is also the case with this version as well.
<img width="600" alt="image" src="https://user-images.githubusercontent.com/1755269/74969157-e0eaab00-53d0-11ea-8c27-1119fdfeafa4.png">